nfd/node-feature-rules-openshift.yaml

```diff
@@ -62,7 +81,7 @@ spec:
         - feature: pci.device
           matchExpressions:
             vendor: {op: In, value: ["8086"]}
-            device: {op: In, value: ["37c8", "4940"]}
```

Comment:
@mregmi is these the device ID for 4xxx QAT device? Can we add some HW detail about the 4xxx QAT device?

Reply:
These are OK. What details you had in mind?

Reply:
All the devices with 4xxx device ID should be the integrated QAT 2.0 HW on SPR or later xeon platforms. @mythi is that correct?